walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -28,11 +28,11 @@
   text-align: center;
 }
 
-.color {
+.changing-color {
   color: blueviolet;
 }
 
-.coloring {
+.styling-titles {
```

Comment:
This is fine for now but "styling" is probably redundant here since this is a css class after all. Just "titles" should be explanatory enough

Reply:
Makes sense, will change that now :)